src/ui/server.js

```diff
@@ -22,25 +23,31 @@ export default function render(
 
   function renderStandardRoute() {
     const bundlePath = locals.assets["main"];
+
+    // Reference for MUI stylesheet injections into SSR HTML:
+    // https://material-ui.com/guides/server-rendering/#handling-the-request
+    const sheets = new ServerStyleSheets();
     const {html, css} = StyleSheetServer.renderStatic(() =>
       ReactDOMServer.renderToString(
-        <StaticRouter location={path} context={context}>
-          <App />
-        </StaticRouter>
+        sheets.collect(
+          <StaticRouter location={path} context={context}>
+            <App />
+          </StaticRouter>
+        )
       )
     );
+
     const page = dedent`\
       <!DOCTYPE html>
       <html>
       <head>
       <meta charset="utf-8">
       <meta name="viewport" content="width=device-width,initial-scale=1" />
       <link rel="shortcut icon" href="${assets.resolve("/favicon.png")}" />
-      <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
-      <link href="https://fonts.googleapis.com/css?family=Roboto+Condensed" rel="stylesheet">
```

Comment:
Is this related? I am happy to drop these, but should we move that to a
different PR?

Reply:
These lines have been un-removed from the pull request